### ISSUE TITLE
VCF? gVCF? GFF? BAM? SAM?

### ISSUE THREAD

**Issue by abetusk** [BioJulia/BioFmtSpecimens#4]
Any plans on adding VCF, gVCF, GFF, BAM, SAM or others?

**Comment by dcjones** [comment]
Yes, absolutely. We're adding things as they get implemented in Bio.jl. It's taken a while to iron out our parsing system, but those will be included in the near future. I'd also be happy to merge contributions as well.
